### DISCUSSION TITLE
[@types/jest] is missing replaceProperty

### DISCUSSION THREAD

**Discussion by ghdoergeloh** [DefinitelyTyped/DefinitelyTyped#64164] [Issues with a @types package]
The function [replaceProperty](https://jestjs.io/docs/jest-object#jestreplacepropertyobject-propertykey-value) is not supported by the types.

[a comment by an automated account is not shown]

**Comment by mrazauskas** [comment]
Related https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64052#issuecomment-1402353804

**Comment by mrazauskas** [comment]
Fix in #64743
